core-api-library/library/src/main/java/net/gini/android/core/api/RetrofitHelper.kt

```diff
@@ -0,0 +1,52 @@
+package net.gini.android.core.api
+
+import net.gini.android.core.api.authorization.UserService
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.util.concurrent.TimeUnit
+
+object RetrofitHelper {
+    private var baseUrl = ""
+    private const val DEFAULT_TIMEOUT = 60L
+
+    val setAuthorizationHeaderUseCase = SetAuthorizationHeaderUseCase()
+
+    fun getInstance(): Retrofit {
+        return Retrofit.Builder()
+            .client(
+                OkHttpClient.Builder()
+                .connectTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS)
+                .readTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS)
+                .writeTimeout(DEFAULT_TIMEOUT, TimeUnit.SECONDS)
+                    .addInterceptor {
+                        it.proceed(setAuthorizationHeaderUseCase(it.request()))
+                    }.build())
+            .baseUrl(baseUrl)
```

Comment:
I just realised that since we work with two apis (user.gini.net and pay-api.gini.net) we will probably need two retrofit instances. We can discuss it next week.